### ISSUE TITLE
ts_library fails when only composed of .d.ts files

### ISSUE THREAD

**Issue by achew22** [bazelbuild/rules_typescript#24]
You get the following error. As a temporary fix you can just include an empty `.ts` file

```
ERROR: /home/achew22/js/externs/BUILD:3:1: in ts_library rule //externs:externs: 
Traceback (most recent call last):
        File "/home/achew22/js/externs/BUILD", line 3
                ts_library(name = 'externs')
        File "/home/achew22/.cache/bazel/_bazel_achew22/b4046dd61443a6da2a0d424d121b9b4a/external/build_bazel_rules_typescript/internal/build_defs.bzl", line 116, in _ts_library_impl
                compile_ts(ctx, is_library = True, compile_acti..., <2 more arguments>)
        File "/home/achew22/.cache/bazel/_bazel_achew22/b4046dd61443a6da2a0d424d121b9b4a/external/build_bazel_rules_typescript/internal/common/compilation.bzl", line 199, in compile_ts                compile_action(ctx, inputs, outputs, tsconfig_json_es...)
        File "/home/achew22/.cache/bazel/_bazel_achew22/b4046dd61443a6da2a0d424d121b9b4a/external/build_bazel_rules_typescript/internal/build_defs.bzl", line 53, in compile_action
                ctx.action(progress_message = "Compiling Ty..., <6 more arguments>)
illegal argument in call to action: action outputs may not be empty.
```

**Comment by alexeagle** [comment]
Interesting, there is really no output to associate with such a compilation action (and nothing for `tsc` to do).
Maybe a set of .d.ts files should just use a https://docs.bazel.build/versions/master/be/general.html#filegroup - but I think we'd need to change something to allow a `filegroup` in the deps of a `ts_library`?

**Comment by fortuna** [comment]
I ran into the same issue.

I was trying to add a `.d.ts` file to the srcs of a `ts_library` rule, but bazel complained that the srcs files have to be all in the same package. Then I turned the .d.st into a library, so I could add to the `deps` section, and then I hit this issue.

It's still unclear to me how one would add a .d.ts dependency to a `ts_library` when it lives in a different directory, which is usually the case.
